buffer/src/test/java/io/netty/buffer/AbstractPooledByteBufTest.java

```diff
@@ -26,49 +24,14 @@
     @Override
     protected ByteBuf newBuffer(int length) {
         ByteBuf buffer = alloc(length);
+
+        // Testing if the writerIndex and readerIndex are correct when allocate and also after we reset the mark.
+        assertEquals(0, buffer.writerIndex());
+        assertEquals(0, buffer.readerIndex());
+        buffer.resetReaderIndex();
+        buffer.resetWriterIndex();
         assertEquals(0, buffer.writerIndex());
         assertEquals(0, buffer.readerIndex());
         return buffer;
     }
-
-    @Test
```

Comment:
@Scottmitch I removed the test as there is no guarantee that the same buffer is returned when we call alloc again now. That said I added the asserts to the `newBuffer(...)`